benchmarks/scripts/clean.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
This file is identical to final.sh

Reply:
This is just a holdover when `main.sh` was way more important. Now I just call the julia file straight out.